lib/ansible/modules/monitoring/uptimerobot.py

```diff
@@ -6,19 +6,12 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
-ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
Please don't remove ANSIBLE_METADATA, it is required.

Reply:
I am sorry, I just moved it.
I moved it to:
https://github.com/Ninjiner/ansible/blob/patch-1/lib/ansible/modules/monitoring/uptimerobot.py#L71

Reply:
`ANSIBLE_METADATA` is kept after `__metaclass__` for all modules so you might want to keep the way it was.

Reply:
I moved it back, where it was.